doc/api/http2.md

```diff
@@ -3418,6 +3427,7 @@ following additional properties:
 [`net.Socket.prototype.ref()`]: net.html#net_socket_ref
 [`net.Socket.prototype.unref()`]: net.html#net_socket_unref
 [`net.connect()`]: net.html#net_net_connect
+[`net.Socket.bufferSize`]: net.html#net_socket_buffersize
```

Comment:
Nit: should go after `[`net.Socket`]: net.html#net_class_net_socket` (in ASCII sort order).

Reply:
Thanks! Done.